src/views/accounts/AccountChangeSecret/Detail/AccountChangeSecretAsset/index.vue

```diff
@@ -45,6 +45,7 @@ export default {
           name: {
             formatter: DetailFormatter,
             formatterArgs: {
+              drawer: true,
               route: 'AssetDetail',
               can: this.$hasPerm('assets.view_asset')
             }
```

Comment:
I would be happy to help you analyze and review this code snippet! Can I know more about it?